### PR TITLE
Auth pages

### PR DESCRIPTION
**Pull request by msdeshmukh009**
**Pages Added**
- [Login](https://deploy-preview-4--orion-quiz.netlify.app/pages/auth-pages/signin.html)
- [Signup](https://deploy-preview-4--orion-quiz.netlify.app/pages/auth-pages/signup.html)
- [Forgot password](https://deploy-preview-4--orion-quiz.netlify.app/pages/auth-pages/forgot-password.html)

### CODE REVIEW

**Review by Chakravarthi0 — APPROVED** [review]
Looks good

File: pages/auth-pages/forgot-password.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        </li>
+
+        <li>
+          <a class="btn btn-primary">Login</a>
+        </li>
+      </ul>
+    </nav>
+    <!-- --------------------------- -->
+
+    <!-- password-assistance-form -->
```
Comment (AbhishekkGautam) [inline pages/auth-pages/forgot-password.html:38]: can we remove the unwanted comments?

**Review by AbhishekkGautam — APPROVED** [review]
Good to go  👍

File: pages/auth-pages/forgot-password.html (hunk 1 above)
Comment (msdeshmukh009) [reply pages/auth-pages/forgot-password.html:38]: Actually, I made those comments intentionally for separation. But thanks for the review.

**Pull request merged**